.github/workflows/build_test_deploy.yml

```
@@ -1,13 +1,238 @@
+  USER_GID: 1000
+  DOCKER_USER: ${{ secrets.DOCKER_USER }}
+  DOCKER_PASS: ${{ secrets.DOCKER_PASS }}
 
 jobs:
-  foo:
-    runs-on: ubuntu-20.04
+  static_frontend_code_checks:
+    runs-on: ubuntu-24.04
     steps:
     - name: Checkout code
       uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)